api/client/rm.go

```diff
@@ -30,6 +30,7 @@ func (cli *DockerCli) CmdRm(args ...string) error {
 
 	var encounteredError error
 	for _, name := range cmd.Args() {
+		name = stripPrefix(name)
```

Comment:
Sorry, I don't get the argument for having our own function, and *even* if we have a good reason for that I believe it should be implemented in terms of `strings.TrimPrefix`.